src/cgame/cg_tutorial.cpp

```diff
@@ -159,6 +159,8 @@ static entityState_t *CG_BuildableInRange( playerState_t *ps, float *healthFract
 	return &es;
 }
 
+#define COLOR_ALARM "^1" // Red
```

Comment:
In orange maybe? 

```suggestion
#define COLOR_ALARM "^8" // Red
```

Otherwise, LGTM

Reply:
Instead of discussing colors, make it a client side cvar and call it a day? In any any cases, LGTM too.

Reply:
> In orange maybe?

Good idea, it may be more readable, done.